lib/scoped_search/definition.rb

```diff
@@ -306,7 +306,8 @@ def default_fields
 
     # Defines a new search field for this search definition.
     def define(*args)
-      Field.new(self, *args)
+      options = args.last.is_a?(Hash) ? args.pop : {}
+      Field.new(self, *args, **options)
     end
```

Comment:
Wouldn't something like this be cleaner? Not sure if it works on old (2.0ish) ruby though

```ruby
def define(*args, **kwargs)
  Field.new(self, *args, **kwargs)
end
```

Reply:
Tbh I can’t remember if there is a good reason not to do it like that. 

If it works on all needed

Reply:
@adamruzicka doesn't work with ruby 3.0.2